mlflow/utils/databricks_utils.py

```diff
@@ -138,6 +143,28 @@ def get_webapp_url():
     return _get_extra_context("api_url")
 
 
+def get_workspace_info_from_dbutils():
+    dbutils = _get_dbutils()
+    if dbutils:
+        context = json.loads(
+            dbutils.notebook.entry_point.getDbutils().notebook().getContext().toJson())
+        workspace_host = context['extraContext']['api_url']
+        workspace_id = context['tags']['orgId']
+        return workspace_host, workspace_id
+
+
+def get_workspace_info_from_databricks_secrets(tracking_uri):
+    profile, path = get_db_info_from_uri(tracking_uri)
```

Comment:
why not just name `path` `key_prefix`?

Reply:
yeah that's better - was carrying over terminology in my head from the URLlib parsing, but there's no need